services/src/main/java/org/keycloak/protocol/saml/SamlService.java

```diff
@@ -568,6 +550,13 @@ protected Response logoutRequest(LogoutRequestType logoutRequest, ClientModel cl
                 AuthenticatedClientSessionModel clientSession = userSession.getAuthenticatedClientSessionByClient(client.getId());
                 if (clientSession != null) {
                     clientSession.setAction(AuthenticationSessionModel.Action.LOGGED_OUT.name());
+
+                    // TODO Warning it is needed to pass unittests but it is an issue if you want to perform authn with aritfact and Logout without as specified in DEP
```

Comment:
Typo: aritfact
I don't understand `without as specified in DEP`. Furthermore, DEP does not need to be mentioned in Keycloak

Reply:
This line was a warning for me. This line is just removed.